### ISSUE TITLE
Consider moving to python3 docker image

### ISSUE THREAD

**Issue by psychok7** [psychok7/django-yadpt-starter#43]
Since its all Debian based moving to https://hub.docker.com/_/python/ should not be very complicated and we will get smaller images

**Comment by pslobo** [comment]
We could possibly experiment with the slim image. I'd stay away from the alpine image for now though.

**Comment by psychok7** [comment]
I will give it a try on one of my active projects first.

The other images are already on Alpine but i don't think it's a good idea for this one to be as well so im gonna try the `python:3.5.3-slim` (3.6.1 has a bug with celery/kombu https://github.com/celery/kombu/issues/675 )

**Comment by psychok7** [comment]
Maybe it's not a good idea to use the slim image:

```
python:slim

This image does not contain the common packages contained in the default tag and only contains the minimal packages needed to run python. Unless you are working in an environment where only the python image will be deployed and you have space constraints, we highly recommend using the default image of this repository.
```

**Comment by psychok7** [comment]
This seems to work fine but i need to test it more.

```
FROM python:3.5.3
ENV PYTHONUNBUFFERED 1

# We need new PostgreSQL Client versions and the one in Debian 8 is old
RUN touch /etc/apt/sources.list.d/pgdg.list
RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main" >> /etc/apt/sources.list.d/pgdg.list
RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -

RUN apt-get update && apt-get install -y \
  build-essential \
  libpq-dev \
  postgresql-client-9.5 \
  libjpeg-dev \
  binutils \
  libproj-dev \
  gdal-bin \
  libxml2-dev \
  libxslt1-dev \
  zlib1g-dev \
  libffi-dev \
  libssl-dev \
  ipython \
  locales -qq

RUN dpkg-reconfigure locales
# Set the locale
RUN locale-gen C.UTF-8
ENV LANG C.UTF-8
ENV LANGUAGE C:en
ENV LC_ALL C.UTF-8
ENV LC_CTYPE C.UTF-8
RUN locale

# Requirements have to be pulled and installed here, otherwise caching won't work
COPY ./requirements /requirements

RUN pip3 install -r /requirements/local.txt

COPY ./compose/django/dev/entrypoint.sh /entrypoint.sh
RUN sed -i 's/\r//' /entrypoint.sh
RUN chmod +x /entrypoint.sh

COPY . /code
WORKDIR /code

ENTRYPOINT ["/entrypoint.sh"]

```

**Comment by psychok7** [comment]
We are now using `python:3.5.3` with Debian 8 as the base image. Once a few bugs are solved we can upgrade to `python 3.6` and Debian 9 when that becomes available
